python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -147,16 +147,20 @@ def __init__(
         sensor_name: Optional[str] = None,
         resources: Optional[Mapping[str, "ResourceDefinition"]] = None,
         definitions: Optional["Definitions"] = None,
+        last_sensor_start_time: Optional[float] = None,
     ):
         from dagster._core.definitions.definitions_class import Definitions
         from dagster._core.definitions.repository_definition import RepositoryDefinition
 
         self._exit_stack = ExitStack()
         self._instance_ref = check.opt_inst_param(instance_ref, "instance_ref", InstanceRef)
-        self._last_completion_time = check.opt_float_param(
+        self._last_tick_completion_time = check.opt_float_param(
```

Comment:
If the property is being renamed to `last_tick_completion_time` seems that the constructor param should be as well